### PR TITLE
Add the possibility to control a subset of linear coordinates in `TSID::SE3Task`

### PR DESCRIPTION
**Pull request by xela-95**
Fix https://github.com/ami-iit/bipedal-locomotion-framework/issues/725

### CODE REVIEW

File: src/TSID/src/SE3Task.cpp [hunk 1]
```diff
@@ -187,8 +195,33 @@ bool SE3Task::initialize(std::weak_ptr<const ParametersHandler::IParametersHandl
...
+    std::vector<bool> mask;
+    if (!ptr->getParameter("mask", mask) || (mask.size() != m_linearVelocitySize))
+    {
+        log()->info("{} [{} {}] Unable to find the mask parameter. The default value is used:{}.",
+                    errorPrefix,
+                    descriptionPrefix,
+                    frameName,
+                    maskDescription);
+    }
+    else
```
Comment (GiulioRomualdi) [inline src/TSID/src/SE3Task.cpp:207]:
```suggestion
    } else
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,9 @@
 All notable changes to this project are documented in this file.
 
 ## [Unreleased]
+
```
Comment (GiulioRomualdi) [inline CHANGELOG.md:5]:
```suggestion
```

File: src/IK/include/BipedalLocomotion/IK/SE3Task.h [hunk 3]
```diff
@@ -66,6 +66,8 @@ class SE3Task : public IKLinearTask, public BipedalLocomotion::System::ITaskCont
 
     static constexpr std::size_t m_spatialVelocitySize{6}; /**< Size of the spatial velocity vector. */
     static constexpr std::size_t m_linearVelocitySize{3}; /**< Size of the linear velocity vector. */
+    static constexpr std::size_t m_angularVelocitySize{3}; /**< Size of the angular velocity vector.
+                                                            */
```
Comment (GiulioRomualdi) [inline src/IK/include/BipedalLocomotion/IK/SE3Task.h:70]:
```suggestion
    static constexpr std::size_t m_angularVelocitySize{3}; /**< Size of the angular velocity vector. */
```

**Review by GiulioRomualdi — COMMENTED** [review]
Thank you for the PR 🎉 
Here you have some minors comment

**Comment by GiulioRomualdi** [comment]
@xela-95 you can also check https://github.com/ami-iit/bipedal-locomotion-framework/actions/runs/6433244451/job/17469917552?pr=738#step:3:40 to remove the whitespaces

**Comment by xela-95** [comment]
> @xela-95 you can also check https://github.com/ami-iit/bipedal-locomotion-framework/actions/runs/6433244451/job/17469917552?pr=738#step:3:40 to remove the whitespaces

Thank you @GiulioRomualdi, I have removed them.

**Comment by Gio-DS** [comment]
(actually @traversaro): @GiulioRomualdi can we merge this? Thanks!

**Review by GiulioRomualdi — APPROVED** [review]

**Comment by traversaro** [comment]
Thanks @GiulioRomualdi !

**Comment by GiulioRomualdi** [comment]
Thank you all for the effort :)

**Pull request merged**